package.json

```diff
@@ -285,6 +285,11 @@
                     "group": "3_appServiceDeployCommands@2"
                 },
                 {
+                    "command": "appService.DeploymentScript",
```

Comment:
What's this change? A bad merge maybe?

DeploymentScript is already listed below